cmd/up.go

```diff
@@ -224,9 +224,14 @@ func (cmd *UpCmd) Run(
 
 	// configure container ssh
 	if cmd.ConfigureSSH {
-		err = configureSSH(devPodConfig, client, cmd.SSHConfigPath, user, workdir,
-			cmd.GPGAgentForwarding ||
-				devPodConfig.ContextOption(config.ContextOptionGPGAgentForwarding) == "true")
+		devPodHome := ""
+		envDevPodHome, ok := os.LookupEnv("DEVPOD_HOME")
```

Comment:
why did we use an env var over a context option?

Reply:
Because the context is stored in the same dir structure, we couldn't read a context option without knowing where DEVPOD_HOME is first